pkgs/development/python-modules/meep/default.nix

```diff
@@ -132,6 +134,7 @@ buildPythonPackage rec {
     # Generate a python test script
     cat > test.py << EOF
     import meep as mp
+    import meep.mpb
```

Comment:
If you are not using that module in the test script, it may be more clear to add `pythonImportsCheck = [ `meep.mpb` ];` instead of importing that one here.